pkg/src/src/main/scala/edu/berkeley/cs/amplab/sparkr/SQLUtils.scala

```diff
@@ -18,6 +15,22 @@ object SQLUtils {
     arr.toSeq
   }
 
+  // A helper to include grouping columns in Agg()
```

Comment:
This method works for now. Could you add a TODO here saying we can get rid of this after the merge (and just call the spark-internal API without reflection)